### PR TITLE
[plugin.video.montreal.greek-tv] 4.0.1

### PR DESCRIPTION
**Pull request by greektv**
### Description
Montreal Greek TV is broadcasting in-house locally-produced programming in the Greek language from the Greek Community of Greater Montreal in Canada.
Version 4.0.1

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.montreal.greek-tv/resources/lib/default.py [hunk 1]
```diff
@@ -33,7 +34,9 @@ def playlinks(url):
         liz = xbmcgui.ListItem(offscreen=True)
         liz.setProperty('isFolder', 'false')
         if url == RADIO_URL :
+            if xbmc.Player().isPlayingVideo() == True : xbmc.executebuiltin("Action(Stop)")
```
Comment (romanvm) [inline plugin.video.montreal.greek-tv/resources/lib/default.py]: `is True`

Comment (Dimitri-Papadopoulos) [reply plugin.video.montreal.greek-tv/resources/lib/default.py]: it was by inattention that I wrote it this way. I replaced it.
Thank you

File: plugin.video.montreal.greek-tv/addon.py [hunk 2]
```diff
@@ -18,7 +18,9 @@
         along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-
+import xbmc
 from resources.lib.default import plugin 
 
+xbmc.executebuiltin("InhibitScreensaver(true)")
+xbmc.executebuiltin("InhibitIdleShutdown(true)")
```
Comment (romanvm) [inline plugin.video.montreal.greek-tv/addon.py]: This is a definite no-go. If you want to disable the screensaver while your addon is running then you can use `try/finally` clause and restore everything as it was in `finally` block. Also an option in your addon setting would be desirable to control this behavior (disabling the screensaver/idle power off).

**Review by romanvm — CHANGES_REQUESTED** [review]

File: plugin.video.montreal.greek-tv/addon.py (hunk 2 above)
Comment (Dimitri-Papadopoulos) [reply plugin.video.montreal.greek-tv/addon.py]: I made the change .
 I couldn't find the way  as you suggested but added condition that : inhibit screensaver/idle power off is set to false when no playing streaming. 
Also added the settings so the user can disable  screensaver/idle power off.

**Comment by greektv** [comment]
see please last change

**Pull request closed without merging**